dev/com.ibm.ws.app.manager/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-    Copyright (c) 2011-2020 IBM Corporation and others.
+    Copyright (c) 2011 IBM Corporation and others.
```

Comment:
You shouldn't remove an ending date from a copyright.  If anything it was supposed to be put back to `2011-2020` or if you are going to change it to change it to `2011, 2020`